pkg/artifactory/resource_artifactory_local_repository_test.go

```diff
@@ -661,3 +662,47 @@ func TestAccLocalAllRepoTypes(t *testing.T) {
 		})
 	}
 }
+
+func makeLocalRepoTestCase(repoType string, t *testing.T) (*testing.T, resource.TestCase) {
+	name := fmt.Sprintf("terraform-local-%s-%d-full", repoType, rand.Int())
+	resourceName := fmt.Sprintf("artifactory_local_%s_repository.%s", repoType, name)
+	repoLayoutRef := getDefaultLocalRepoLayoutRef(repoType, true)
```

Comment:
So this function:
`getDefaultLocalRepoLayoutRef` I didn't see it implemented in this PR - when was this created?

Reply:
variable packageType used to replace literal `docker` from all occurrences

Reply:
Following 4 functions were implemented.
```
getDefaultLocalRepoLayoutRef
getDefaultFederatedRepoLayoutRef
getDefaultVirtualRepoLayoutRef
getDefaultRemoteRepoLayoutRef
```
As per the code **improvement**, all the above 4 functions has been removed from `repositories.go` file and a new function `func getDefaultRepoLayoutRef(repositoryType string, packageType string) string` has been implemented.